jpdb_anki_import/importer.py

```diff
@@ -45,7 +45,7 @@ def create_note(self, vocab: jpdb.Vocabulary) -> Note:
 
     def card_state_current_next(self, card: Card, rating: str) -> (CardAnswer, CardAnswer):
         # The following code is taken directly from the Anki v3 scheduler
-        states = self.anki.col._backend.get_next_card_states(card.id)
+        states = self.anki.col.backend.get_scheduling_states(card.id)
```

Comment:
Looks like this name was changed in this commit: https://github.com/ankitects/anki/commit/e39fb74e82d98964c25a2a7f38e61a17c77dabf7. It might be nice to support somewhat older versions of Anki, though, so what if we did something like this? I tested this out on Anki 2.1.54.
```suggestion
        if hasattr(self.anki.col.backend, 'get_scheduling_states'):
            states = self.anki.col.backend.get_scheduling_states(card.id)
        else:
            # Anki < 2.1.60
            states = self.anki.col.backend.get_next_card_states(card.id)
```

Reply:
Good idea (I also didn't know you could do that in Python). I tested this and it works on Anki 2.1.51 and 2.1.52 so I set the minimum version to 2.1.51 in the manifest.

If you ever aim to support older versions in the future, I also found that 2.1.45 (oldest version with V3 scheduler) fails to load the add-on on startup, and 2.1.50 loads the add-on properly, but the cards themselves don't import (stalled at 0 cards imported).